src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -509,6 +511,12 @@ public string RoleClaimType
             set;
```

Comment:
nit: should app properties use 1 line instead this?

Reply:
We seem to be inconsistent about this, though I'm not sure which format is preferred...

Reply:
not sure either... but seemed like "default" properties seemed to follow the "collapsed" guidelines. 

---
In reply to: [285785710](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1179#discussion_r285785710) [](ancestors = 285785710)

Reply:
I created an issue to track this so we can address it in the future: #1186